app/bundles/CampaignBundle/Command/TriggerCampaignCommand.php

```diff
@@ -208,9 +208,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->limiter = new ContactLimiter($batchLimit, $contactId, $contactMinId, $contactMaxId, $contactIds, $threadId, $maxThreads, $campaignLimit);
 
         defined('MAUTIC_CAMPAIGN_SYSTEM_TRIGGERED') or define('MAUTIC_CAMPAIGN_SYSTEM_TRIGGERED', 1);
-
         $id = $input->getOption('campaign-id');
-        if (!$this->checkRunStatus($input, $this->output, $id)) {
+
+        $moderationKey = ($id ? 'id:'.$id : '').($threadId ? 't:'.$threadId : '');
```

Comment:
Probably this would be more elegant:

        `$key    = sprintf('%s-%s', $id, $threadId);`